eng/Xunit3/Xunit3.targets

```diff
@@ -0,0 +1,18 @@
+<Project>
```

Comment:
@Youssef1313 is there a reason this isn't done at the Arcade level?

Reply:
WIP: https://github.com/dotnet/arcade/pull/15668

Reply:
But you will need to wait until you are using latest Arcade 10 anyways

Reply:
Awesome!